packages/client/src/workflow-client.ts

```diff
@@ -809,7 +810,7 @@ export class WorkflowClient {
           searchAttributes: mapFromPayloads(
             searchAttributePayloadConverter,
             raw.workflowExecutionInfo!.searchAttributes?.indexedFields
```

Comment:
Doesn't have to be in this PR but let's get rid of all of these non null assertions